src/ports/postgres/modules/summary/summary.sql_in

```diff
@@ -285,6 +294,16 @@ string should be double-quoted; in this case the input would be '"MyTable"').
     in a slow but exact method. The most frequent values are computed using a
     faithful implementation that preserves the approximation guarantees of
     the Cormode/Muthukrishnan method (more information in \ref grp_mfvsketch).
+- Summary statistics are calculated for each grouping 
```

Comment:
We're repeating this text in the argument list and in notes. That could be fine, but checking if it's intentional?